### DISCUSSION TITLE
Continuous Reboot Issue with Security Onion (Awaiting for reboot)

### DISCUSSION THREAD

**Discussion by Chizan-Dolrocks** [Security-Onion-Solutions/securityonion#14050] [Unsupported Versions]
I have encountered an issue with Security Onion (version 2.4.110). The system continuously prompts for a reboot and displays the status "**Reboot**", making it unusable.

### **Screenshot of the Error:**

![awaiting for reboot](https://github.com/user-attachments/assets/bb5e7bd8-c959-4dad-8f90-d3cba3a59c08)

### **Additional Information:**

**Version**: Security Onion 2.4.110
**Main Symptom**: The system remains stuck on the "Reboot" status, requiring constant restarts.

### **Request for Assistance:**

I am looking for a solution to resolve this issue and restore Security Onion's normal operation. Any help or suggestions would be greatly appreciated!

**Comment by rh-ops** [comment]
Those reboot requests are generally from kernel updates to the base OS.  It's not necessary to reboot right way if you see that message, it's just reminder that reboots are needed for those kernel updates, which can be applied during your normal update or change management windows.
